ivy/utils/backend/sub_backend_handler.py

```diff
@@ -12,7 +12,6 @@
 _sub_backend_dict = dict()
 _backend_to_sub_backends_dict = dict()
 
-
```

Comment:
Let's revert this line here. lint-fixes shouldn't be part of this PR.

Reply:
Fixed